saleor/product/models.py

```diff
@@ -311,7 +311,7 @@ class Product(SeoModel, ModelWithMetadata):
     description = SanitizedJSONField(
         blank=True, default=dict, sanitizer=clean_editor_js
     )
-    description_plaintext = TextField(blank=True, default="")
+    description_plaintext = TextField(blank=True)
```

Comment:
Is this related?

Reply:
This is difference between 2.11 and 3.0, i removed it to keep consistency 2.11 and 3.0. This changes doesn't affect on functionality